src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/UserProfileService.java

```diff
@@ -70,17 +78,18 @@ public SubUser load(String principal) throws Exception {
           throw new JaggaerApplicationException(INTERNAL_SERVER_ERROR.value(),
               "Invalid state: Jaggaer company profile data must contain exactly 1 'GURU' record");
         }
-
-        Set<SubUser> subUsers = getCompanyDataResponse.getReturnCompanyData().stream().findFirst()
-            .get().getReturnSubUser().getSubUsers();
+        ReturnCompanyData returnCompanyData =
+            getCompanyDataResponse.getReturnCompanyData().stream().findFirst().get();
```

Comment:
Minor point - SonarLint is recommending using a `var` and also `Optional#isPresent()`